.github/workflows/on_push.yml

```diff
@@ -143,11 +143,52 @@ jobs:
         name: "Integration tests"
         run: make test-integration
 
+  version:
+    name: Version
+    concurrency: tagging
+    if: ${{ github.ref == 'refs/heads/release' || github.ref == 'refs/heads/develop' }}
+    runs-on: ubuntu-latest
+    needs: [
+      test,
+      test-integration,
+      test-windows,
+      test-integration-windows,
+      test-macos,
+      test-integration-macos]
+    outputs:
+      semver_tag: ${{ steps.semver-tag.outputs.semver_tag }}
+      ancestor_tag: ${{ steps.semver-tag.outputs.ancestor_tag }}
+      is_prerelease: ${{ steps.semver-tag.outputs.is_prerelease }}
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Calculate semver tag
+        id: semver-tag
+        uses: wakatime/semver-action@v1.3.2
+        with:
+          prerelease_id: alpha
+          main_branch_name: release
+      - name: Create tag
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ github.token }}
+          script: |
+            github.git.createRef({
```

Comment:
This creates a lightweight tag, but we've been creating an annotated tag indirectly via Releases. Let's use `github.git.createTag` like https://github.com/wakatime/wakatime-cli/pull/427/files#diff-53dbc06610f897f66670a951aca3d54295c4a0fa85d7869295330aa05265a761R179?

Reply:
Unless creating the release "upgrades" the lightweight tag to an annotated tag?

Reply:
None of our tags are annotated. Running the following command tells if tag is annotated or not. `git for-each-ref refs/tags`:
<img width="592" alt="Screen Shot 2021-05-29 at 18 33 03" src="https://user-images.githubusercontent.com/782854/120085202-256cf800-c0a4-11eb-9338-95f0c851351b.png">

Docs says:
```
[T]he output has three fields: The hash of an object, the type of the object, and the name in refs/tags that refers to the object. A so-called "lightweight" tag is a name in refs/tags that refers to a commit¹ object. An "annotated" tag is a name in refs/tags that refers to a tag object.
```
Do you want to make then annotated instead?

Reply:
No, let's use lightweight tags. My mistake.